2-mandatory/1-create-functions.js

```diff
@@ -33,7 +37,10 @@ Write a function that:
 - Returns a new array containing the same elements, but without the element at the passed index.
 */
 
-function remove() {
+function remove(arr, index) {
```

Comment:
Hi Shayan, can you explain this code please?

Reply:
I mean the whole function

Reply:
we have an array, we copy that array to a new array:
`const arr2 = arr.slice();`
or you write:
`const newArr = arr`
now we want to remove one element from our new array with the index number which we will get from our function, So here we can use splice:
`arr2.splice(index, 1);`